tutorials/terraform/data-transfer/data-transfer-mmy-objs.tf

```diff
@@ -0,0 +1,160 @@
+# Infrastructure for the Yandex Cloud Data Streams, Object Storage and Data Transfer.
+#
+# RU: https://cloud.yandex.ru/docs/data-transfer/tutorials/mmy-objs-migration
+# EN: https://cloud.yandex.com/en/docs/data-transfer/tutorials/mmy-objs-migration
+#
+# Set source Managed Service for MySQL cluster and target Object Storage bucket settings.
```

Comment:
Set the source Managed Service for MySQL cluster and target Object Storage bucket settings.